### ISSUE TITLE
Function to detect Pixman version throws Exception

### ISSUE THREAD

**Issue by ghadjigeorghiou** [TissueImageAnalytics/tiatoolbox#251]
* TIA Toolbox version: 1.0.0
* Python version: 3.7
* Operating System: MacOS Big Sur

### Description

Toolbox throws an exception for the pixman version.

### What I Did

```
Exception in thread Thread-1:

Traceback (most recent call last):
File "/opt/miniconda3/envs/isyntax2zarr/lib/python3.7/threading.py", line 926, in _bootstrap_inner
self.run()
File "/opt/miniconda3/envs/isyntax2zarr/lib/python3.7/threading.py", line 870, in run
self._target(*self._args, **self._kwargs)
File "/opt/miniconda3/envs/isyntax2zarr/lib/python3.7/site-packages/tiatoolbox/utils/env_detection.py", line 283, in _show_warning
version, using = pixman_version()
File "/opt/miniconda3/envs/isyntax2zarr/lib/python3.7/site-packages/tiatoolbox/utils/env_detection.py", line 249, in pixman_version
version = tuple(int(part) for part in matches.group(1).split("."))
AttributeError: 'NoneType' object has no attribute 'split'
```

**Comment by shaneahmed** [comment]
Thanks @ghadjigeorghiou  
@John-P  please can you check this?

**Comment by John-P** [comment]
@ghadjigeorghiou  How are you running the toolbox? Is open slide installed via homebrew, macports, anaconda, or manually?

**Comment by ghadjigeorghiou** [comment]
It's from anaconda

**Comment by John-P** [comment]
For posterity sake, the issue with the setup in this case was that there was an anaconda environment set up with dependencies installed. However, openslide (and cairo, then pixaman as dependencies in the tree) were not installed in the conda environment bur rather via homebrew. Additonally, brew had two versions of pixman from different dependants installed.

**Comment by John-P** [comment]
@shaneahmed Can you please confirm how you have installed openslide? Was it manually installed on Windows?

**Comment by shaneahmed** [comment]
It was installed as in the README. The binaries are copied to a Windows path. The openslide-python is installed via pip.

**Comment by shaneahmed** [comment]
Fixed in this PR #266
